code/controllers/subsystem/processing/quirks.dm

```diff
@@ -98,7 +109,7 @@ PROCESSING_SUBSYSTEM_DEF(quirks)
 	//Create a random list of stuff to start with
 	while(bonus_quirks > added_quirk_count)
 		var/quirk = pick(possible_quirks) //quirk is a string
-		if(quirk in quirk_blacklist) //prevent blacklisted
+		if(quirk in GLOB.quirk_blacklist) //prevent blacklisted
```

Comment:
`quirk_blacklist` is a list of lists so this check does nothing

So this code has probably always been broken forever